eval.py

```diff
@@ -20,6 +20,8 @@
 
 import logging
 
+from yahoo_fin import stock_info as si
```

Comment:
Not sure what this import is for

Reply:
In order to get a live price for stocks, I use yahoo finance in order to verify I have enough for that stock, if that want there, the code would fail every 3 stocks.